jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST

```diff
@@ -11,9 +11,11 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalpost" -c "base ocnanalpost"
 ##############################################
 
 export COMOUT=${COMOUT:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/ocean}
+export COMOUTice=${ROTDIR}/${CDUMP}.${PDY}/${cyc}/ice
 export CDATE=${CDATE:-${PDY}${cyc}}
 
-mkdir -p "${COMOUT}"
+mkdir -p "${COMOUT}/RESTART"
+mkdir -p "${COMOUTice}/RESTART"
```

Comment:
I'm puzzled about this - was it not making a `RESTART` directory before (for the ocean)?

Reply:
... Ha! good catch, I don't need line 17. I need line 18 to save the cice restart that contains the analsyis.